src/include/btmem.h

```diff
@@ -127,11 +127,12 @@ __wt_page_header_byteswap(WT_PAGE_HEADER *dsk)
  */
 struct __wt_addr {
     /* Validity window */
-    wt_timestamp_t newest_durable_ts;
     wt_timestamp_t oldest_start_ts;
     uint64_t oldest_start_txn;
+    wt_timestamp_t start_durable_ts;
     wt_timestamp_t newest_stop_ts;
     uint64_t newest_stop_txn;
+    wt_timestamp_t newest_durable_ts; /* Known in 4.4 as stop_durable_ts */
```

Comment:
Shouldn't this match the second group in `WT_CELL_UNPACK`? But the names are different?

Reply:
I've changed them to match.  I was trying to "reuse" the old field name to avoid source code changes, but I see that's confusing and the source code changes are not very extensive.